src/main/java/com/nekomaster1000/infernalexp/entities/VolineEntity.java

```diff
@@ -80,5 +215,51 @@ public boolean isImmuneToFire() {
         return true;
     }
 
+    @OnlyIn(Dist.CLIENT)
+    public boolean isEating() {
+        return isEating;
+    }
+
+    @OnlyIn(Dist.CLIENT)
+    @Override
+    public void handleStatusUpdate(byte id) {
+        if (id == 8) {
+            isEating = false;
+        } else if (id == 9) {
+            isEating = true;
+        } else {
+            super.handleStatusUpdate(id);
+        }
+    }
+    public static class VolineEatItemsGoal extends EatItemsGoal<VolineEntity> {
```

Comment:
Just a thematic suggestion, not needed
```suggestion
    }
    
    public static class VolineEatItemsGoal extends EatItemsGoal<VolineEntity> {
```